classes/form/CustomerAddressForm.php

```diff
@@ -68,8 +68,23 @@ public function fillWith(array $params = [])
 
     public function validate()
     {
+        $validated = true;
+
         if (!parent::validate()) {
-            return false;
+            $validated =  false;
+        }
+
+        if (($city = $this->getField('city'))) {
```

Comment:
Yes, that would be a nice feature.

Making the validation depend on the ObjectModel's `$definition` would be nice